docs/clients/p4rt-ctl.rst

```diff
@@ -91,6 +91,24 @@ services that are available for a p4runtime client.
 For information on security and enabling TLS on gRPC ports, refer to
 guides/security-guide.md
 
+Usage: p4rt-ctl [OPTIONS] COMMAND [ARG...]
+positional arguments:
+  command               Subcommand to run
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -g GRPC_ADDR, --grpc_addr GRPC_ADDR P4Runtime gRPC server address, format : <server IP>:<port>
```

Comment:
Is the .rst rendering of document ok for this line? In GitHub, it looks like the line does not have enough whitespace and does not align with previous line

Reply:
Fixed it and now it's seen as
<img width="360" alt="image" src="https://github.com/ipdk-io/networking-recipe/assets/22621220/d9f62ec7-161e-46f5-a573-53c9340aab22">